Exercise/compute_lowest_terms.py

```diff
@@ -1,17 +1,19 @@
 ## TODO: complete the function "lowest_terms" below
 from fractions import Fraction
 def lowest_terms(x):
```

Comment:
- function missing docstring

     review distance_to_destination() function in Example folder for sample docstring

- comment your code appropriately

- commits should use 'active voice' eg. 
   'add case to handle....'  
   not 
  'adding case .....'